style.css

```diff
@@ -257,7 +257,7 @@ header i {
   display: none;
   margin-top: 7px;
   display: none;
-  width:50px;
+  min-width:150px;
 }
```

Comment:
Using pixels not the best way to set width and sizes

Reply:
Dont use pixels

Reply:
done